src/BaseTemplates/ClassLibrary/project.json

```diff
@@ -1,21 +1,14 @@
 ﻿{
-  "version": "1.0.0-*",
-  "description": "$projectname$ Class Library",
-  "authors": [ "$username$" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
+    "version": "1.0.0-*",
+    "compilationOptions": {
+        "emitEntryPoint": false
+    },
 
-  "frameworks": {
-    "net451": { },
-    "dotnet5.4": {
-      "dependencies": {
-        "Microsoft.CSharp": "4.0.1-*",
-        "System.Collections": "4.0.11-*",
-        "System.Linq": "4.0.1-*",
-        "System.Runtime": "4.0.21-*",
-        "System.Threading": "4.0.11-*"
-      }
+    "dependencies": {
+        "NETStandard.Library": "1.0.0-rc2-23810"
+    },
+
+    "frameworks": {
+        "dotnet5.5": { }
```

Comment:
Given @Pilchie's comment on `netstandard1.0` being the final TFM, it may be better to have this be `dotnet5.1` since it's the same thing.  Sorry for the incorrect suggestion about `dotnet5.5`!